src/loaders/compute_tools/gtdb_tk/gtdb_tk.py

```diff
@@ -122,14 +131,37 @@ def _run_gtdb_tk(
             f"Missing IDs {error_tool_safe_data_ids} that are not found in either "
             f"{GTDB_FILTER_FILE_PATTERN} or {GTDB_FAIL_GENOME_FILE}"
         )
-    
+
     for miss_tool_safe_data_id in miss_tool_safe_data_ids:
         error_message = filtered_or_failed_genome_id_mapping[miss_tool_safe_data_id]
         fatal_tuple = create_fatal_tuple(miss_tool_safe_data_id, ids_to_files, error_message)
         fatal_tuples.append(fatal_tuple)
 
     write_fatal_tuples_to_dict(fatal_tuples, output_dir)
 
+    _process_gtdb_result(output_dir, ids_to_files, summary_files)
+
+
+def _process_gtdb_result(
+        output_dir: Path,
+        ids_to_files: Dict[str, GenomeTuple],
+        summary_files: List[str],
+):
+    genome_id_col = loader_common_names.GTDB_GENOME_ID_COL
+    gtdb_tk_docs = list()
+    for tool_file_name in summary_files:
+        docs = read_genome_attri_result(
+            output_dir,
+            tool_file_name,
+            SELECTED_GTDBTK_SUMMARY_FEATURES,
+            genome_id_col,
+            ids_to_files)
```

Comment:
Does this need to set the check file flag to false since both summary files aren't necessarily there?

Reply:
I looked at the code again. The func that returns gtdb_tk result files actually will only return the ones generated. So the file will guaranteed to be there. 
https://github.com/kbase/collections/blob/main/src/loaders/compute_tools/tool_common.py#L604-L607
Do you want me to remove the check flag or just leave it there? It doesn't hurt to just leave it there I think.

Reply:
I think it's fine as is